src/UnityNuGet/RegistryEntry.cs

```diff
@@ -15,6 +15,9 @@ public class RegistryEntry
         [JsonProperty("listed")]
         public bool Listed { get; set; }
 
+        [JsonProperty("scope")]
+        public string? UnityScope { get; set; }
```

Comment:
I understand this has to do with https://github.com/xoofx/UnityNuGet/issues/216 so I think you have mixed this PR with a different issue, no?

Also I already argued why we can't/shouldn't introduce this behavior.

Reply:
I removed that, sorry